django_project/changes/templates/version/detail.html

```diff
@@ -21,7 +21,7 @@ <h1>Entries (all)</h1>
     <div class="row" style="margin-top:15px;margin-bottom:15px;">
         <div class="col-lg-12">
             <div class="btn-group btn-group pull-left">
-                {% if user.is_authenticated and user.is_staff %}
+                {% if user_can_view and user_can_edit or user_can_delete %}
```

Comment:
Maybe we should do only one condition in each 'if' like `user_is_logged_in`, `user_is_changelog_manager`, `user_is_staff`. So you would have only one condition everytime in the template

Reply:
Thanks @Gustry 
I have updated the checks, I removed used a single variable